packages/eas-cli/src/credentials/ios/actions/AscApiKeyUtils.ts

```diff
@@ -94,7 +94,6 @@ async function generateAscApiKeyAsync(
   ctx: CredentialsContext,
   purpose: AppStoreApiKeyPurpose
 ): Promise<MinimalAscApiKey> {
-  //console.log('debug');
```

Comment:
👀 Lame, polish devs use `console.log('dupa');` 🤣  (this is true)

Reply:
^ That's correct, this methodology is officially called DDD (Dupa-driven debugging) 😆

Reply:
😠 😠 😠 😠 😠 😠 😠 😠 😠 
![Screen Shot 2022-04-06 at 11 46 21 AM](https://user-images.githubusercontent.com/6380927/162047093-aa90a98d-1125-45c0-a007-9dcddbb646d8.png)